Components/Avatar/avatar.css

```diff
@@ -0,0 +1,32 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.avatar {
+  max-inline-size: 100%;
+  block-size: auto;
+  border-radius: 50%;
+  height: 5rem;
```

Comment:
remove height, width from general avatar class as it is included in avatar-md class